server/php-master.cpp

```diff
@@ -1520,28 +1513,29 @@ WorkerType run_master() {
 
     me->generation = generation;
 
-    if (to_kill != 0 || to_run != 0 || job_workers_to_kill != 0 || job_workers_to_run != 0) {
-      vkprintf(1, "[to_kill = %d] [to_run = %d] [job_workers_to_kill = %d] [job_workers_to_run = %d]\n", to_kill, to_run, job_workers_to_kill, job_workers_to_run);
+    if (general_worker_to_kill != 0 || general_worker_to_run != 0 || job_workers_to_kill != 0 || job_workers_to_run != 0) {
+      tvkprintf(master_process, 2, "[general_worker_to_kill = %d] [general_worker_to_run = %d] [job_workers_to_kill = %d] [job_workers_to_run = %d]\n",
+                general_worker_to_kill, general_worker_to_run, job_workers_to_kill, job_workers_to_run);
     }
 
     for (int i = 0; i < job_workers_to_kill; ++i) {
       kill_worker(WorkerType::job_worker);
     }
     for (int i = 0; i < job_workers_to_run; ++i) {
       if (run_worker(WorkerType::job_worker)) {
-        tvkprintf(job_workers, 1, "launched new job worker with pid = %d\n", pid);
```

Comment:
why did you remove it?

Reply:
move this info inside `run_worker(...)` on 564 and 655 lines